crates/core_simd/examples/nbody.rs

```diff
@@ -1,11 +1,17 @@
-#![cfg_attr(feature = "std", feature(portable_simd))]
+#![feature(portable_simd)]
+
+extern crate std_float;
 
 /// Benchmarks game nbody code
 /// Taken from the `packed_simd` crate
 /// Run this benchmark with `cargo test --example nbody`
-#[cfg(feature = "std")]
 mod nbody {
-    use core_simd::*;
+    #[cfg(not(feature = "std"))]
+    use core::simd::*;
+    #[cfg(feature = "std")]
+    use core_simd::simd::*;
```

Comment:
what's going on here? shouldn't it always use `core_simd`--that's what we're testing

Reply:
Oh yeah. 🙃 Just was doing the same impl flagging dance everywhere. Thanks! (it... shouldn't really matter in practice.)